class  B3
{
	int test()
	{
		try
		{
			
		}
		catch(ArithmeticException ex)
		{
			return 20;
		}
	}
}
